server/webhook_parser.go

```diff
@@ -245,7 +247,7 @@ func parseWebhookCommentCreated(jwh *JiraWebhook) (Webhook, error) {
 		JiraWebhook: jwh,
 		eventTypes:  NewStringSet(eventCreatedComment),
 		headline:    fmt.Sprintf("%s **commented** on %s", commentAuthor, jwh.mdKeySummaryLink()),
-		text:        truncate(quoteIssueComment(jwh.Comment.Body), 3000),
+		text:        truncate(quoteIssueComment(preProcessText(jwh.Comment.Body)), 3000),
```

Comment:
Do we need to do this for only the comments?

Reply:
We are adding it only to the updated events. What about issue update/created/deleted events?